test/integration/targets/zypper_repository/tasks/zypper_repository.yml

```
@@ -125,3 +125,10 @@
     priority: 100
     auto_import_keys: true
     state: "present"
+
+- name: add test repo with params
+  zypper_repository:
+    name: "test"
+    repo: http://dl.google.com/linux/chrome/rpm/stable/x86_64
+    state: present
+    extra_args_precommand: --root {{ output_dir | expanduser }}/testdir/
```

Can we have an `assert` for that test?
The current state of the test would pass even if you never used `extra_args_precommand` anywhere.

Maybe
```suggestion
    extra_args_precommand: --root {{ output_dir | expanduser }}/testdir/

- name: check listrepo for custom root
  command: zypper --root {{ output_dir | expanduser}}/testdir/ lr test
  register: zypper_result

- assert:
    that:
      - '"dl.google.com" in zypper_result.stdout'
```

---

Done